docs/plugins/building_a_plugin/_layer_data_guide.md

```diff
@@ -56,6 +56,11 @@ class ArrayLike(Protocol):
 # typing than literal array types (e.g. numpy, dask, xarray, etc...)
 ```
 
+Note that when writing your own plugin, such type annotations are nearly always optional, 
+except in rare cases like `magicgui` function widgets.
+Here, for clarity, we've provided aliases that indicate how values of the type are 
+being used and the level of generality that's possible (e.g., with `ArrayLike`). 
```

Comment:
Sorry I don't quite follow, what is meant by aliases here? Thanks

Reply:
Sorry, I just meant different names (with clearer context meaning) for the types. I changed here: 5ce242db122766b936b151ef328e7c65b7d19d5d
WDYT?